RockWeb/Blocks/Crm/BulkUpdate.ascx.cs

```diff
@@ -46,7 +47,8 @@ namespace RockWeb.Blocks.Crm
     [Description( "Used for updating information about several individuals at once." )]
 
     [AttributeCategoryField( "Attribute Categories", "The person attribute categories to display and allow bulk updating", true, "Rock.Model.Person", false, "", "", 0 )]
-    [IntegerField( "Display Count", "The initial number of individuals to display prior to expanding list", false, 0, "", 1  )]
+    [IntegerField( "Display Count", "The initial number of individuals to display prior to expanding list", false, 0, "", 1 )]
+    [WorkflowTypeField( "Workflow Actions", "The workflows to make available for bulk updating.", true, false, "", "", 2 )]
```

Comment:
Can you change the name/key of this setting to "Workflow Types". (instead of "Workflow Actions").